plopfile.js

```diff
@@ -82,4 +65,37 @@ module.exports = function(plop) {
       ];
     },
   });
+
+  plop.setGenerator('tsconfig', {
+    description: 'Generate package level tsconfig',
+    prompts: [],
+    actions(data) {
+      const packageNames = getPackageNames();
+
+      return [
+        {
```

Comment:
can we also make this inject the cross-package dependencies? We should be able to simply compile, for each package, a list of `@shopify/<pkg>` dependencies, then filter that by the `packageNames` variable and generate the `references` section from that data.

Reply:
That does seem like it would make this less cumbersome, I'll see if I can think of any downsides to that approach